doc/compiler/optimizer/BenefitInliner.md

```diff
@@ -0,0 +1,94 @@
+# BenefitInliner
+
+BenefitInliner introduces a novel way of inlining. Compared to the traditional way of inlining relies on hard-coded heuristics, BenefitInliner provides a systematic way of evaluating inlining candidates that combines the dynamic and static benefits of inlining. 
+
+## Overview of main phases of BenefitInliner
+1. Build the search space for candidate methods to be inlined.
+    1. **IDTBuilder** constructs an inlining dependency tree (**IDT**).
+    2. **AbstractInterpreter** generates an **InliningMethodSummary** stashing potential optimization opportunities and specifying the constraints for parameters to make optimizations happen.
+    3. **AbstractInterpreter** computes abstract state values for each program state. 
```

Comment:
I don't understand this sentence. Is "each program state" supposed to be "each program point"?

Reply:
Yes, it should be ""each program point"